i18n/locales/en_us.json

```diff
@@ -0,0 +1,30 @@
+{
+  "welcome": "American, Welcome!",
+  "shared": "Shared between all en locales",
+  "userHeader": {
+    "links": {
+      "store": "Store",
+      "library": "Library",
+      "community": "Community",
+      "news": "News"
+    },
+    "notifications": {
+      "unread": "Unread Notifications",
+      "all": "View all →",
+      "none": "No notifications"
+    },
+    "profile": {
+      "signout": "Signout",
+      "admin": "Admin Dashboard",
+      "settings": "Account settings"
+    }
+  },
+  "store": {
+    "recentlyAdded": "Recently Added",
+    "lookAt": "Check it out",
+    "noGame": "no game",
+    "recentlyReleased": "Recently released",
+    "exploreMore": "Explore more →",
```

Comment:
Does i18n resolve HTML entity references? We should probably try to keep these as `&rarr;` if possible.

Reply:
It does not like them one bit, they get rendered as raw strings if we do that. IDK how to keep it as the reference

Reply:
Seems like we need to turn off escaping for it to work. Honestly doesn't seem that dangerous because we control the input.

Reply:
I can't find a config option for that, can you link it?

Reply:
It seems like it's only a thing in `react-i18next`: https://github.com/i18next/react-i18next/pull/1538